tools/hicup/hicup_hicup.xml

```diff
@@ -110,7 +110,7 @@
                 <param name="own_file" value="chr1_pipe.fa"/>
             </conditional>
             <!-- outputs -->
-            <output name="hicup_results" file="hicup_results_pipe.html" compare="sim_size" delta="500"/>
+            <output name="hicup_results" file="hicup_result_pipe.html" compare="sim_size" delta="500"/>
```

Comment:
HiCUP gives for the same input slightly different results for each run. This makes it very difficult to compare and to test.